SecurityPkg/Include/Library/Tcg2InitEventLib.h

```diff
@@ -0,0 +1,59 @@
+/** @file
+  Initializes TCG event data structures
+
+Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+**/

```

Comment:
You can consider adding `MU_CHANGE` tags to the top of the file. It sometimes helps connect the pieces.